packages/lesswrong/components/tagging/FooterTagList.tsx

```diff
@@ -32,7 +32,7 @@ const styles = (theme: ThemeType) => ({
     marginBottom: 8,
   } : {
     display: 'flex',
-    gap: '4px',
+    gap: 4,
```

Comment:
I'm pretty sure this isn't a valid `gap` value, and it explicitly needs to be a string with a `px` suffix

Reply:
I know that used to be true but it seems to be working now.

Reply:
🤷

separately, and this doesn't super matter in any individual instance, but when it's not clear what's motivating a change like this, it's costlier to review, and when it's actually a no-op that really suggests not making them (or leaving them in).